base/base-builder/Dockerfile

```diff
@@ -19,10 +19,15 @@ FROM ubuntu:yakkety
 
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt-get update
```

Comment:
How it appear in you PR? Is it your change? Is it related to port up/down task?

Reply:
Its not part of port up/down task, its actually appears when we rebase with develop branch and above changes done by Eugene Frolov #1087

Reply:
@gauravchug , please fix this. They should not be a part of the pull request.